docker/standalone-ha.xml

```diff
@@ -118,6 +113,7 @@
                 <!--   xxxxxxxxxxxxxxxxxxxxxxxxx   -->
                 <handlers>
                     <handler name="CONSOLE"/>
+                    <handler name="FILE"/>
```

Comment:
Let's remove the creation of a file handler for the logging. It takes a lot of resources from KC which already uses a lot. We don't make use of the FILE handler but let's avoid its creation.